src/test/ui/keyword_generics/const_trait.stderr

```diff
@@ -0,0 +1,102 @@
+error[E0308]: mismatched effects
+  --> $DIR/const_trait.rs:29:5
+   |
+LL |     A::bar();
+   |     ^^^^^^^^ expected `const`, found `!const`
+   |
+   = note: expected effect `const`
+              found effect `!const`
```

Comment:
This doesn't seem to be good UI, but given that this is an unstable feature, I don't think this is blocking.

Reply:
opinions on the new errors?

This is still the fallback error, but I haven't been able to actually trigger it.

Reply:
Seems fine to me. As long as this doesn't get leaked to stable users I'm okay with it.